dlt/common/storages/versioned_storage.py

```diff
@@ -48,7 +48,8 @@ def version(self) -> semver.VersionInfo:
         return self._load_version()
 
     def _load_version(self) -> semver.VersionInfo:
-        return self.storage.load(VersionedStorage.VERSION_FILE)
+        version_str = self.storage.load(VersionedStorage.VERSION_FILE)
+        return semver.VersionInfo.parse(version_str)
```

Comment:
@rudolfix I had to change this to it passes mypy check after I updated poetry lock. Let me know if this is the proper way to do this.

Reply:
maybe semver got update and they have typings now. perfect!